mcx/jax.py

```diff
@@ -105,7 +102,7 @@ def progress_bar_scan(func):
         def wrapper_progress_bar(carry, x):
             "x is a tuple: (iteration_number, key)"
             iter_num = x[0]
-            _ = _update_progress_bar(iter_num, iter_num)
+            _update_progress_bar(iter_num)
```

Comment:
@rlouf you're right it doesn't need to return anything :)

I also simlified the function: it only needs to take in a single argument

Reply:
Nice!